corehq/ex-submodules/casexml/apps/phone/tests/test_ota_restore.py

```diff
@@ -232,7 +230,9 @@ def get_all_syncslogs():
         restore_payload = deprecated_generate_restore_payload(
             self.project, self.restore_user, items=items)
 
-        sync_log_id = get_last_synclog_for_user(self.restore_user.user_id).get_id
+        sync_log_id = SyncLogSQL.objects.filter(
+            user_id=self.restore_user.user_id
+        ).order_by('date').last().synclog_id.hex
```

Comment:
Can you explain this change? I assumed you were just removing the unused function in this commit but looks like you redefined its use here.

Reply:
this was the only usage of the function and it's in test code so rather than leave the function I implemented it's functionality here.

Reply:
earlier this was calling `get_id` on a wrapped doc but now you are doing `synclog_id.hex`. I could not connect that change.

Reply:
It comes from here: https://github.com/dimagi/commcare-hq/blob/9c1efdf758fdb26da591b4b1f77cb351a2177d22/corehq/ex-submodules/casexml/apps/phone/models.py#L388-L389